test/cli/template-test/local/in/my-template/v0/tests/two/expected-out/.keboola/manifest.json

```diff
@@ -0,0 +1,99 @@
+{
+  "version": 2,
+  "project": {
+    "id": %%TEST_KBC_PROJECT_ID%%,
+    "apiHost": "%%TEST_KBC_STORAGE_API_HOST%%"
+  },
+  "sortBy": "id",
+  "naming": {
+    "branch": "{branch_name}",
+    "config": "{component_type}/{component_id}/{config_name}",
+    "configRow": "rows/{config_row_name}",
+    "schedulerConfig": "schedules/{config_name}",
+    "sharedCodeConfig": "_shared/{target_component_id}",
+    "sharedCodeConfigRow": "codes/{config_row_name}",
+    "variablesConfig": "variables",
+    "variablesValuesRow": "values/{config_row_name}"
+  },
+  "allowedBranches": [
+    "*"
+  ],
+  "ignoredComponents": [],
+  "templates": {
+    "repositories": [
+      {
+        "type": "dir",
+        "name": "keboola",
+        "url": "../repository",
+        "ref": "main"
+      }
+    ]
+  },
+  "branches": [
+    {
+      "id": %%LOCAL_STATE_MAIN_BRANCH_ID%%,
```

Comment:
@webrouse tady mám asi problém. 🙂 Tohle má být test k templatě v E2E testu a to nahrazení tohoto placeholderu by se mělo provést tady někde https://github.com/keboola/keboola-as-code/blob/7d0d034688e407206bb7066506eacefe555dbcc9/pkg/lib/operation/template/local/test/operation.go#L193. Ale ono se provede už v `cli_test.go`, že? 🤔

Reply:
Měl jsem tady `%%TEST_BRANCH_MAIN_ID%%` a pořád mi to podivně failovalo. Teď jsem to změnil na ten `%% LOCAL_STATE_MAIN_BRANCH_ID%%` který používám ve fixture projektu (https://github.com/keboola/keboola-as-code/blob/7d0d034688e407206bb7066506eacefe555dbcc9/internal/pkg/fixtures/local/empty-branch/.keboola/manifest.json#L28) a už mi failuje E2E test že tu promšnnou nedokáže nahradit. 
![image](https://user-images.githubusercontent.com/215660/180265598-f2ee692c-12ae-4ff4-a51a-cf85fd40f5c9.png)
Tak teď přemýšlím co s tím, musím tam dát nějaký placeholder (resp. user bude muset). Nebo by stačilo jen `%s`?

Reply:
1. Teda robis "testovanie testovania", vystup nie je problem porovna `wildcards.Assert` nahradzuje `%%` -> `%`: https://github.com/keboola/go-utils/blob/be395f3a33c1a01a57d7b42f060663369880caad/pkg/wildcards/wildcards.go#L113

2. Rovnako mozes upravit aj vstup, napr. pouzit `\%\%` na vstupe, ... regexp to bude ignorovat, a na zaver nahradis vsetky `\%\%` -> `%%`: https://github.com/keboola/keboola-as-code/blob/1da27e1d3799b8eb3e0f41c6533195c8d16470ff/internal/pkg/utils/testhelper/env.go#L23-L29

Reply:
TLDR: staci pridat nejaku formu escapovania, aby tie envs/wildcards vykonal az testovany program.

Reply:
Uzivatatel tento problem mat nebude, lebo to nebude spustat cez nase e2e testy.

Reply:
Ak riesis, ze uzivatel bude musel riesit branch_id, tak navrhujem dat tam napevno, do toho prazdneho manifestu, pevne `branchId=1`.

Reply:
To jsem zkoušel dát tam pevný branch id, jenže do actual stavu se dostane odněkud reálný id, myslím že do manifestu z operace použití šablony jestli si dobře pamatuju.

Reply:
Když tam dám tu jedničku: https://github.com/keboola/keboola-as-code/pull/752/commits/2fccd5b4425b5209a4018daa3c794a2273502036 tak to začne fialovat na:
![CleanShot 2022-07-25 at 09 04 38@2x](https://user-images.githubusercontent.com/215660/180717861-dc0b2cd5-c94d-4455-b737-0826b875cafa.jpg)


https://github.com/keboola/keboola-as-code/runs/7494809227?check_suite_focus=true